### DISCUSSION TITLE
Práctica 1

### DISCUSSION THREAD

**Discussion by Darry96** [Launch-X-Latam/MisionFrontEnd#116] [Q&A]
Para la práctica donde dice que la página debe ser responsiva para un móvil es que solo debemos hacer el diseño en movil o también hay que hacerla en web?

**Comment by maxkario** [comment]
Yo creo que debería ser web ya que estamos en la parte de frontend. Pero siempre hay que hacer un diseño responsive que va orientado a varios tipos de pantallas. Mas adelante veremos esos conceptos.

**Reply by Darry96** [reply]
Es que mi duda es porque en las instrucciones de la práctica dice que debe ser responsiva para un móvil por eso mi incógnita

**Reply by SeleneAlmaraz** [reply]
Responsiva significa que debe ser vista de manera correcta tanto como en la página web tanto como en el celular, dentro de la consola de tu navegador puedes jugar con el tamaño y el comportamiento de tu pagina para las distintas pantallas si das click en este símbolo
![image](https://user-images.githubusercontent.com/114202057/193432352-16e29d16-a101-4083-b87c-7d99afdccd18.png)
para entrar a la consola solo debes dar ctr+i

**Reply by Darry96** [reply]
Pero ahorita nomás estamos con el diseño de la página no? O también debemos ir haciendo el código ?

**Reply by maxkario** [reply]
Solo los bocetos. Como una especie de maquetación web pero en papel o en su defecto Figma o Illustrator.

**Comment by samujm** [comment]
Hola, siempre que se hace un desarrollo web es necesario tener en cuenta las tres medidas (computadora, tableta y celular), en este caso solo están solicitando tener en cuenta que nuestro sistema web lo quieren visualizar en un teléfono, te recomiendo leer un poco del paradigma mobile first para darte una idea de cómo puedes pasar elementos de un entorno móvil a una computadora de escritorio y como sugerencia te diría que intentes realizar la practica con ese cambio de tamaños para aprendizaje personal. Saludos.

**Reply by Darry96** [reply]
Ok no conocía ese concepto lo acabo de buscar y encontré varios artículos mañana en mi tiempo libre los repasare muchas gracias y buena noche 👍
